pkg/persistence/config/writer/config_writer_test.go

```diff
@@ -1164,11 +1167,101 @@ func TestWriteConfigs(t *testing.T) {
 				"general/alerting-profile/a.json",
 			},
 		},
+		{
+			name: "Documents",
+			configs: []config.Config{
+				{
+					Template: template.NewInMemoryTemplateWithPath("project/dashboard/a.json", ""),
+					Coordinate: coordinate.Coordinate{
+						Project:  "project",
+						Type:     "dashboard",
+						ConfigId: "configId1",
+					},
+					Type: config.DocumentType{
+						Type: config.DashboardType,
+					},
+					OriginObjectId: "ext-ID-123",
+					Parameters: map[string]parameter.Parameter{
+						config.NameParameter: &value.ValueParameter{Value: "name"},
+					},
+					Skip: true,
+				},
+				{
+					Template: template.NewInMemoryTemplateWithPath("project/notebook/a.json", ""),
+					Coordinate: coordinate.Coordinate{
+						Project:  "project",
+						Type:     "notebook",
+						ConfigId: "configId2",
+					},
+					Type: config.DocumentType{
+						Type: config.NotebookType,
+					},
+					OriginObjectId: "ext-ID-123",
+					Parameters: map[string]parameter.Parameter{
+						config.NameParameter: &value.ValueParameter{Value: "name"},
+					},
+					Skip: true,
+				},
+			},
+			expectedConfigs: map[string]persistence.TopLevelDefinition{
+				"dashboard": {
+					Configs: []persistence.TopLevelConfigDefinition{
+						{
+							Id: "configId1",
+							Config: persistence.ConfigDefinition{
+								Name:           "name",
+								Parameters:     nil,
+								Template:       "a.json",
+								OriginObjectId: "ext-ID-123",
+								Skip:           true,
+							},
+							Type: persistence.TypeDefinition{
+								Type: config.DocumentType{
+									Type: "dashboard",
+								},
+							},
+						},
+					},
+				},
+				"notebook": {
+					Configs: []persistence.TopLevelConfigDefinition{
+						{
+							Id: "configId2",
+							Config: persistence.ConfigDefinition{
+								Name:           "name",
+								Parameters:     nil,
+								Template:       "a.json",
+								OriginObjectId: "ext-ID-123",
+								Skip:           true,
+							},
+							Type: persistence.TypeDefinition{
+								Type: config.DocumentType{
+									Type: "notebook",
+								},
+							},
+						},
+					},
+				},
+			},
+			expectedTemplatePaths: []string{
+				"project/dashboard/a.json",
+				"project/notebook/a.json",
+			},
+			envVars: map[string]string{
+				featureflags.Documents().EnvName(): "true",
+			},
+		},
 	}
 
+	t.Setenv(featureflags.Documents().EnvName(), "true")
```

Comment:
There is the filed `envVars map[string]string`! Why not set it there?

Reply:
Correct, this line should be removed